cypress/integration/RadioGroup.spec.ts

```diff
@@ -0,0 +1,40 @@
+import * as h from '../helpers';
+
+const getInidivualRadio = (index: number) => {
+  return cy
+    .get('div[name="contact"]')
+    .children()
+    .eq(index)
+    .children()
+    .eq(0)
+    .children()
+    .eq(0);
+};
```

Comment:
if there's a better way to do this, I'm open to suggestions!

Reply:
This can probably be simplified, I believe jQuery has the ability to return a collection of `radio`s using a selector. Or there's always just `'div[name="contact"] input[type="radio"]'` which should get you all the radios in a collection.

Thoughts on making the `name` configurable too or add a parameter that takes a custom selector string since `name` is technically optional. I could easily see us renaming the `name` in the story and breaking the helper for everyone